examples/nrf-rtos-trace/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+edition = "2021"
+name = "embassy-nrf-rtos-trace-examples"
+version = "0.1.0"
+
+[features]
+default = ["log", "nightly"]
+nightly = ["embassy-executor/nightly", "embassy-nrf/nightly", "embassy-nrf/unstable-traits"]
+log = [
+    "dep:log",
+    "embassy-util/log",
+    "embassy-executor/log",
+    "embassy-time/log",
+    "embassy-nrf/log",
+]
+
+[dependencies]
+embassy-util = { version = "0.1.0", path = "../../embassy-util" }
+embassy-executor = { version = "0.1.0", path = "../../embassy-executor", features=["rtos-trace", "rtos-trace-interrupt", "integrated-timers"] }
+embassy-time = { version = "0.1.0", path = "../../embassy-time" }
+embassy-nrf = { version = "0.1.0", path = "../../embassy-nrf", features = ["nrf52840", "time-driver-rtc1", "gpiote", "unstable-pac"] }
+
+cortex-m = { version = "0.7.6", features = ["critical-section-single-core"] }
+cortex-m-rt = "0.7.0"
+panic-probe = { version = "0.3" }
+futures = { version = "0.3.17", default-features = false, features = ["async-await"] }
+rand = { version = "0.8.4", default-features = false }
+serde = { version = "1.0.136", default-features = false }
+rtos-trace = "0.1.3"
+systemview-target = { version = "0.1.1", features = ["callbacks-app", "callbacks-os", "log", "cortex-m"] }
+log = { version = "0.4.17", optional = true }
+
+[[bin]]
+name = "rtos_trace"
+required-features = ["nightly"]
+
+[patch.crates-io]
+rtos-trace = { git = "https://gitlab.com/quentinmit/rtos-trace.git", branch = "build-fix" }
```

Comment:
I saw this got merged into the upstream repo, so it's probably good to change this now to avoid accidentally breaking at some later point (and preferrably point it to a specific rev).

Reply:
Yeah, I'm hoping they'll do a new release soon and I can remove the patch entirely.

Reply:
This now compiles with no crate patches. If that's the only remaining comment, can we merge this now?